pkg/commands/deps.go

```diff
@@ -34,6 +34,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
```

Comment:
Had to replicate these to avoid imports.

Reply:
This feels like enough reason to finally delete `ko deps`, which has been hidden and deprecated for a while.

Reply:
Ah, didn't know! Shall I do this in this PR or follow up?

Reply:
Update: removed